package.json

```diff
@@ -36,9 +36,9 @@
     "@commitlint/travis-cli": "8.2.0",
     "husky": "4.2.5",
     "local-web-server": "3.0.7",
-    "node-sass": "4.12.0",
     "npm-run-all": "4.1.5",
     "rimraf": "3.0.0",
+    "sass": "1.49.9",
```

Comment:
I strongly support the change from `node-sass` to `dart-sass` as it is the standard & reference implementation and `node-sass` is deprecated for quite some time now.